cmd/cardinality.go

```diff
@@ -232,6 +257,35 @@ func (m *seriesTable) updateWhileBrowsingTable(msg tea.Msg) (tea.Model, tea.Cmd)
 		case "up":
 			m.table, cmd = m.table.Update(msg)
 			return m, cmd
+		case "enter", "v":
+			selectedRow := m.table.SelectedRow()
+			metricName := selectedRow[0]
+			seriesText := m.seriesScrapeText[metricName]
+
+			tmpFile := internal.CreateTempFileWithContent(seriesText)
+			if tmpFile == "" {
+				return m, m.flashMsg.Flash("Failed to create temporary file", internal.Error, flashDuration)
+			}
+
+			editor := os.Getenv("EDITOR")
+			if editor == "" {
+				os.Remove(tmpFile)
+				return m, m.flashMsg.Flash("Please set the EDITOR environment variable", internal.Error, flashDuration)
+			}
+
+			// Run the editor and wait for it to complete
+			cmd := exec.Command(editor, tmpFile)
+			cmd.Stdin = os.Stdin
+			cmd.Stdout = os.Stdout
+			cmd.Stderr = os.Stderr
+
+			// Run the editor and clean up
+			err := cmd.Run()
+			os.Remove(tmpFile)
```

Comment:
I think it is fine to leave those temp files. If you try to open on a GUI editor, the file will be empty. I tried with `EDITOR=code`.